fast64_internal/f3d/f3d_material.py

```diff
@@ -2440,6 +2442,61 @@ def reloadDefaultF3DPresets():
             update_preset_manual_v4(material, presetNameToFilename[material.f3d_mat.presetName])
 
 
+class UpdateColorSpacePopup(Operator):
+    bl_label = "Update Color Space"
+    bl_idname = "dialog.update_colorspace"
```

Comment:
I can't easily look at other fast64 operators to check what convention we use but the bl_idname should be less generic, like throw in "fast64" in there